api/swagger.yaml

```diff
@@ -800,6 +800,31 @@ paths:
         default:
           $ref: '#/responses/UnknownError'
 
+  /control/newDiscoveredAPIs:
+    post:
+      summary: 'List of new external services discovered by API gateways'
```

Comment:
The description should say, this method allows a client (a gateway for example) to notify APIclarity about newly discovered APIs.

In the description I would also  make sure it's clear that this can be a cumulative list or delta, which means that if the list contains already existing APIS, then they are ignored (or updated?)